hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleToken.java

```diff
@@ -573,6 +574,11 @@ public void setAccountsFrozenByDefault(boolean accountsFrozenByDefault) {
 		this.accountsFrozenByDefault = accountsFrozenByDefault;
 	}
 
+	public void setAccountsKycGrantedByDefault(boolean accountsKycGrantedByDefault) {
```

Comment:
nit: the parameters could be made `final` where possible.

Reply:
Thanks @abhishek-hedera, Will update them in the main PR for branch `ci-enable-2143` - PR #2221